Python/syntax_test_python.py

```diff
@@ -1490,70 +1490,80 @@ class Starship:
 # Type comments - type: ignore must be by itself.
 
 primes = 5  # type: ignore # type: not-a-type-comment
```

Comment:
Just for clarification

```
primes = 5  # type: ignore # type: str   // technically ok but weird
primes = 5  # type: str  # type: ignore  // not tested here but typical
primes = 5  # type: str  # type: str     // not tested here and invalid
```

(I don't know how fine-grained these tests are here.)

Reply:
The impression I had gotten from the previous implementation was that line comments added into a type comment effectively ended the processing of mypy. Is that true?

From your comment above, it almost sounds like mypy will process nested comments that start with `type:` and the only logical combinations are `type: ignore` and a normal `type:` comment, right?

Reply:
Maybe mypy stops after the first `type: ignore` comment. 🤷  After that everything is "egal".  The only thing you see in practice is `# type: str  # type: ignore` to brutally force a type.  And typical `# type: ignore[foo-rule]  # mypy bug https:\\issue-nr` because you really get hit by a mypy bug every other day.